src/App.tsx

```diff
@@ -12,14 +12,17 @@ import { RunAllButton } from './components/RunAllButton/RunAllButton';
 import { DropdownSelect } from './components/DropdownSelect/DropdownSelect'
 
 const URL = "https://kroger-description-api-0b391e779fb3.herokuapp.com/"
+const wordLists = {
```

Comment:
What if we have this list live in the express app and get requested by the frontend when it needs it? Maybe we can institute some sort of `init` call to the backend that includes moving the "auth" logic to the backend as well with the passkey living there. We can talk about it on a call soon.